data/json/npcs/epilogues.json

```
@@ -311,5 +461,155 @@
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_34",
+    "text": "     Depressed, she joined a protestant community and became a paragon of virtue.  Devoting her life to bible study, she spent the rest of her life in relative peace."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_35",
+    "text": "     Killed and devoured by a cannibal just a few days after your death, hardly anyone remembers her name."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_36",
+    "text": "     Became a renowned merchant after your death, she built her own outpost in the west.  An avid book collector, she established a great library to keep the flame of knowledge burning."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_37",
+    "text": "     Travelled around for months before finding an old farm.  She spent a few years tending to her crops and living the life of a hermit before being killed by raiders for fun."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_38",
+    "text": "     Travelled south and joined a settlement there.  She became a renowned engineer, building the largest steam engine in New England."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_39",
+    "text": "     Joined a gang after your death, she spent months preying on the innocent, before dying in a dispute with a fellow brute."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_40",
+    "text": "     Devoured alive by a group of zombies, she kept thinking of you as she died."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_41",
+    "text": "     She found an abandoned APC and spent years travelling around and collecting personal stories.  She wrote one of the few post-apocalyptic books, detailing the life after the end."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_42",
+    "text": "     Kept wandering for years. Her life was cut short when a group of raiders used her as a target practice."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_43",
+    "text": "     She became a skilled game hunter and one of the best shots in the wasteland.  Years later she joined the Old Guard and was given the position of Marshal."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_44",
+    "text": "     Desperate to save humanity from extinction, she organised her own group of survivors, whose job was to hunt bandits and decorate roads with their mutilated bodies.  She died in a skirmish a few months later."
+  },
+  {
+    "type": "epilogue",
+    "group": "female",
+    "id": "npc_female_45",
+    "text": "     She became an arsonist but burned alive a few weeks later."
```

I think this could be improved (it's somewhat brief) - here a few of my tries.
```suggestion
    "text": "     She became an arsonist, but burned alive in a fire she set a few weeks later."
```
```suggestion
    "text": "     She became an arsonist, and was incinerated a few weeks later in a fire she set."
```
```suggestion
    "text": "     She became an arsonist but burned alive a few weeks later, trying to escape a horde."
```